### PR TITLE
new: usr: Add SQL UDF example function & correct javascript UDF example [FLYCS-984]

### PR DESCRIPTION
**Pull request by yannis-arg**

### CODE REVIEW

File: docs/functions.rst [hunk 1]
```diff
@@ -4,12 +4,18 @@ Functions
 
 Flycs lets you define BigQuery functions. These functions will be created in BigQuery during the CI/CD pipeline and usable in your transformations.
 
-Example definition of a function in YAML:
+.. note::
+   Javascript functions using external code libraries are currently not supported.
+
+   If you still want to use javascript UDF importing external packages in your Flycs
+   transformation, you can create them in the raw dataset & reference it from this dataset.
```
Comment (christophedcpm) [inline docs/functions.rst]: I don't get this. Why the raw dataset ? And I also do not want to have special behavior *when PII module is deactivated* in the documentation. We should have a solution that works for the normal setup of Flycs.

**Review by christophedcpm — CHANGES_REQUESTED** [review]

File: docs/functions.rst (hunk 1 above)
Comment (yannis-arg) [reply docs/functions.rst:11]:
```suggestion
   Javascript functions using external code libraries are not supported.
```

File: docs/functions.rst [hunk 2]
```diff
@@ -4,12 +4,18 @@ Functions
 
 Flycs lets you define BigQuery functions. These functions will be created in BigQuery during the CI/CD pipeline and usable in your transformations.
 
-Example definition of a function in YAML:
+.. note::
+   Javascript functions using external code libraries are currently not supported.
```
Comment (yannis-arg) [inline docs/functions.rst:8]:
```suggestion
   Javascript functions using external code libraries are currently not supported.
```

File: docs/functions.rst (hunk 1 above)
Comment (yannis-arg) [reply docs/functions.rst]: Doc file changed to only inform JS external feature is not supported. (https://github.com/devoteamgcloud/flycs_sdk/pull/38/commits/0f8180e935ca9d179d2ce1a788e136181aca9068)
I also removed that from the forum (see https://forum.flycs.io/t/user-manual-concepts-function/85)

Comment (christophedcpm) [reply docs/functions.rst]: Ok good, let's keep it that way until FLYCS-989 is solved.

**Review by christophedcpm — APPROVED** [review]
LGTM

**Pull request merged**